### ISSUE TITLE
GMLRS rockets are wrongly called GMLRS ER

### ISSUE THREAD

**Issue by gosnold** [PygmalionOfCyprus/cmo-db-requests#1519]
### DB Selector

DB3K

### Affected DBID(s)

2699, 50, 3136

### Summary of Changes

They are called GMLRS ER in the DB but are the standard range versions with 45nm. ER versions were developed later and have 150km range.

So rename them to GMLRS

Also the designations are wrong, the unitary (2699) is designated M31, not M30A1

Finally the weapon codes are wrong. Both should have Ballistic Missile or Rocket Booster, Weapon - INS w/ GNSS Navigation

and not level cruise flight that they currently have

Targets should be 

Surface Vessel, Land Structure - Soft, Radar, Mobile Target - Soft, Mobile Target - Personnel

For alternative warhead and

Surface Vessel, Land Structure - Soft, Land Structure - Hardened, Runway, Radar, Mobile Target - Soft, Mobile Target - Hardened

for unitary

### Sources

manufacturer: https://www.lockheedmartin.com/en-us/products/guided-mlrs-unitary-rocket.html

**Comment by PygmalionOfCyprus** [comment]
Fixed for 495. Though re: targets, I'll note that "Radar" specifically refers to emitting radars (e.g. for ARMs); the weapon's ability to target the radar facility itself is covered by the Land Structure and/or Mobile Target capabilities (depending on whether the radar is a static building or a mobile unit).

**Comment by gosnold** [comment]
Ah good catch, sorry about that.

**Comment by PygmalionOfCyprus** [comment]
Just sticking my head back in to say I doubled back while on an unrelated research kick and drastically improved modeling of the alternative warhead. For some reason (probably a bad copy base) it was being modeled as an anti-tank fragmentary munition, effective even against "Special" armor, which was just silly when you look at what the AW is actually capable of.

I reworked the warhead based mostly on [this article](https://www.army.mil/article/180724/new_munitions_replace_cluster_bomb_rounds_that_pose_danger_to_civilians); from 495 it'll be treated as a 90kg fragmentation warhead, effective against unarmored targets but basically useless against anything else.
